### PR TITLE
Branch one

### PR DESCRIPTION
**Pull request by BereketRetta**
This PR is for changing the js file of the Awesome_books project to ES6 syntax.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```
Comment (mikethreels) [inline yarn.lock:3]: - [x] according to the requirements you should Use `npm init -y` command to create a package.json file. something seems to have gone wrong as there is no package.json file.

File: node_modules/.yarn-integrity [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [
+    "whiskeyshop"
+  ],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```
Comment (mikethreels) [inline node_modules/.yarn-integrity:12]: - [x] according to the GENERAL REQUIREMENTS, the node_modules folder should be added to the .gitignore file.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  localBooks,
+} from './modules/variables.js';
+
+// import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+import renderBook from './modules/render.js';
+import setStore from './modules/setStore.js';
+
+const { luxon } = window;
+const date = document.getElementById('date');
+date.innerText = luxon.DateTime.now().toFormat('MMMM dd, yyyy, hh:mm:ss a').toString();
```
Comment (mikethreels) [inline main.js:14]: - [x] due to the background colour and the colour of the date/time being the same this is not visible, please make sure to change the colour of the date/time so it is visible for the user without having to select the text.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import {
+  renderBooks,
+  titleInput,
+  authorInput,
+  addBook,
+  localBooks,
+} from './modules/variables.js';
+
+// import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
```
Comment (mikethreels) [inline main.js]: - [x] In order to keep your codebase clean please remove any commented out code.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Bereket, 

Good job changing the functions to arrow functions 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @BereketRetta,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**